.github/workflows/build.yaml

```diff
@@ -60,6 +60,10 @@ jobs:
       # those targets first and only builds the package if they succeed.
       # Build the sdist and wheel distribution of the package and docs as a zip file.
       run: make dist
+    # Audit all currently installed packages for security vulnerabilities.
+    - name: Audit installed packages
+      if: matrix.os == env.ARTIFACT_OS && matrix.python == env.ARTIFACT_PYTHON
```

Comment:
Why not to always run `make audit`?

Reply:
Sure, why not: commit https://github.com/jenstroeger/python-package-template/pull/307/commits/fb448419629edeb0e5ea152e3110a1c7b8e44af3.